src/src_V2/app/App.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
```

Comment:
I think it is not a good idea to create src_V2 folder, maybe we need to discuss another solution.

Reply:
We discussed to have this folder and do coding there, did you mean the name or the entire idea which we have discussed?

Reply:
@sasuntitan said to create a "modules" folder and separate every module on it, am I correct?

Reply:
I think no, you can double check our conversation in slack.